core/components/molecules/emptyState/EmptyStateImage.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { BaseProps } from '@/utils/types';
+import EmptyStateContext from './EmptyStateContext';
+import classNames from 'classnames';
+
+export interface EmptyImageProps extends BaseProps, React.ImgHTMLAttributes<HTMLImageElement> {
+  /**
+   * The content to be displayed within the component.
+   * Can be any renderable React Node as Image or Icon.
+   */
+  children?: React.ReactNode;
+
+  /**
+   * Source URL of the image to be displayed.
+   * This is used as the `src` attribute for an `img` tag when there are no children provided.
+   * If `children` are provided, `src` is ignored and the children content is displayed instead.
+   */
+  src?: string;
+
+  /**
+   * The height of the image.
+   * It defines the vertical size of the image element and accepts both numeric and string values.
+   * If not defined, the image height will be determined by the original image size or by the CSS of the image container.
+   */
+  height?: string | number;
+
+  /**
+   * The minimum height of the image.
+   */
+  minHeight?: string | number;
+
+  /**
+   * The maximum height of the image.
+   */
+  maxHeight?: string | number;
+}
+
+export const imageHeight = {
+  standard: '200px',
+  compressed: '150px',
+  tight: '100px',
+  large: '200px',
+  small: '200px',
+};
+
+const EmptyStateImage = (props: EmptyImageProps) => {
+  const { children, maxHeight, height, minHeight, src, className, ...rest } = props;
+  const contextProp = React.useContext(EmptyStateContext);
+
+  const imageWrapperClasses = classNames(className);
+
+  const { size = 'standard' } = contextProp;
+
+  const sizeStyle = {
+    maxHeight: maxHeight ?? imageHeight[size],
+    height: height,
+    minHeight: minHeight,
+  };
+
+  if (children) {
+    return (
+      <div {...rest} className={imageWrapperClasses} style={{ ...sizeStyle }}>
+        {children}
+      </div>
+    );
+  }
+
+  return (
+    <>
+      {src && (
+        <div>
+          {
+            // TODO(a11y)
+            //eslint-disable-next-line
+          <img
+              className={`${imageWrapperClasses} w-100 h-100`}
```

Comment:
Instead of separately adding `w-100 h-100` classes please add them inside `imageWrapperClasses`

Reply:
done